config.py

```diff
@@ -82,194 +102,288 @@
     RegDeleteValue.restype = LONG
     RegDeleteValue.argtypes = [HKEY, LPCWSTR]
 
-    def KnownFolderPath(guid):
+    def known_folder_path(guid):
+        """Look up a Windows GUID to actual folder path name."""
         buf = ctypes.c_wchar_p()
         if SHGetKnownFolderPath(ctypes.create_string_buffer(guid.bytes_le), 0, 0, ctypes.byref(buf)):
             return None
-        retval = buf.value	# copy data
-        CoTaskMemFree(buf)	# and free original
+        retval = buf.value  # copy data
+        CoTaskMemFree(buf)  # and free original
         return retval
 
-
-elif platform=='linux':
+elif platform == 'linux':
     import codecs
     from configparser import RawConfigParser
 
 
 class Config(object):
```

Comment:
```suggestion
class Config:
```

No need to subclass object as we dont support py2 anymore